components/BtnContainer/BtnContainer.js

```diff
@@ -0,0 +1,20 @@
+import React, {PropTypes} from 'react';
+
+import css from './BtnContainer.css';
+
+export default class BtnContainer {
+  render() {
+    const {children, className, ...props} = this.props;
+
+    return (
+      <button {...props} className={[css.root, className].join(' ')}>
```

Comment:
There\s currently no style in the css though so perhaps worth removing it as an option until it's needed?